src/polyglot-notebooks-vscode-common/src/commands.ts

```diff
@@ -229,12 +232,34 @@ export function registerFileCommands(context: vscode.ExtensionContext, parserSer
             newLanguageOptions.push(languageName);
         }
 
+        const polyglotConfig = vscode.workspace.getConfiguration(constants.PolyglotConfigurationSectionName);
+        if (useDefaults) {
+            // try to get the default notebook language/kernel
+            const defaultLanguageName = polyglotConfig.get<string>('defaultNotebookLanguage');
+            if (defaultLanguageName) {
+                await newNotebookWithLanguage(viewType, defaultLanguageName, false);
+                return;
+            }
+        }
+
         const notebookLanguage = await vscode.window.showQuickPick(newLanguageOptions, { title: 'Default Language' });
         if (!notebookLanguage) {
             return;
         }
 
         const kernelName = languagesAndKernelNames[notebookLanguage];
+        await newNotebookWithLanguage(viewType, kernelName, useDefaults);
+    }
+
+    async function newNotebookWithLanguage(viewType: string, kernelName: string, saveDefault: boolean | undefined): Promise<void> {
+        // save the default
+        if (saveDefault) {
+            // If we got here, the user entered via a path that expected to use a default, but it wasn't present.  Show
+            // an info box letting them know that a default can be specified.  The result isn't awaited because they'll
+            // likely want to continue with their current action.
+            promptToSaveConfigValue(`Would you like all future new notebooks to be created with the '${kernelName}' language?`, kernelName, constants.PolyglotConfigurationSectionName, 'defaultNotebookLanguage');
```

Comment:
```suggestion
            promptToSaveConfigValue(`Would you like future notebooks to default to the '${kernelName}' kernel?`, kernelName, constants.PolyglotConfigurationSectionName, 'defaultNotebookLanguage');
```

Reply:
Not sure about my wording either so I'll explain what I'm getting at. The kernel name isn't the language name. We could go the other way from my suggestion as well and say e.g. `C# Script language.`

The word "all" seemed a little strong, since the setting can always be changed, and it doesn't add necessary information.

Reply:
How do we feel about "Would you like future notebooks to start with __kernel language__ by default?"